libr/anal/p/anal_evm_cs.c

```diff
@@ -64,19 +64,46 @@ static int analop(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *buf, int len,
 		goto beach;
 	}
 	if (mask & R_ANAL_OP_MASK_DISASM) {
-		char *str = r_str_newf ("%s%s%s", insn->mnemonic, insn->op_str[0]? " ": "", insn->op_str);
+		char *str = r_str_newf ("%s%s%s", insn->mnemonic, insn->op_str[0]? " 0x": "", insn->op_str);
 		op->mnemonic = str;
 	}
+	// Handle https://github.com/capstone-engine/capstone/pull/1231. Can be removed when merged.
+	if (strlen(insn->op_str) / 2 + 1 != insn->size) {
```

Comment:
Uhm dont see the relationship between instruction size and text length :?

what about comparing sizeof(insn.bytes) and inst.size with 24